IIPS/iip-1.md

```diff
@@ -0,0 +1,218 @@
+
+## Transferring IIP Ownership
+
+It occasionally becomes necessary to transfer ownership of IIPs to a new champion. In general, we'd like to retain the original author as a co-author of the transferred IIP, but that's really up to the original author. A good reason to transfer ownership is because the original author no longer has the time or interest in updating it or following through with the IIP process, or has fallen off the face of the 'net (i.e. is unreachable or isn't responding to email). A bad reason to transfer ownership is because you don't agree with the direction of the IIP. We try to build consensus around an IIP, but if that's not possible, you can always submit a competing IIP.
+
+If you are interested in assuming ownership of an IIP, send a message asking to take over, addressed to both the original author and the IIP editor. If the original author doesn't respond to email in a timely manner, the IIP editor will make a unilateral decision (it's not like such decisions can't be reversed :)).
+
+## IIP Editors
+
+The current IIP editors are
+
+` * Jonghyup Kim (@extendjh)`
+
+`* Sojin Kim (@sojinkim-icon)`
+
+`* Jaechang Namgoong (@sink772)`
+
+## IIP Editor Responsibilities
+
+For each new IIP that comes in, an editor does the following:
+
+- Read the IIP to check if it is ready: sound and complete. The ideas must make technical sense, even if they don't seem likely to get to final status.
+- The title should accurately describe the content.
+- Check the IIP for language (spelling, grammar, sentence structure, etc.), markup (Github flavored Markdown), code style
+
+If the IIP isn't ready, the editor will send it back to the author for revision, with specific instructions.
+
+Once the IIP is ready for the repository, the IIP editor will:
+
+- Assign an IIP number (generally the PR number or, if preferred by the author, the Issue # if there was discussion in the Issues section of this repository about this IIP)
+
+- Merge the corresponding pull request
+
+- Send a message back to the IIP author with the next step.
+
+Many IIPs are written and maintained by developers with write access to the ICON codebase. The IIP editors monitor IIP changes, and correct any structure, grammar, spelling, or markup mistakes we see.
+
+The editors don't pass judgment on IIPs. We merely do the administrative & editorial part.
+
+## History
+
+This document was derived heavily from [Ethereum's EIP-1][https://github.com/ethereum/EIPs] which in turn was derived from [Bitcoin's BIP-0001][https://github.com/bitcoin/bips] and [Python's PEP-0001][https://www.python.org/dev/peps/]. In many places text was simply copied and modified. Authors of EIP-1, BIP-0001, and PEP-0001 are not responsible for its use in the ICON Improvement Process, and should not be bothered with technical questions specific to ICON or the IIP. Please direct all comments to the IIP editors.
```

Comment:
Again, link formats should be updated: there are 3 links to be updated.
[https://github...] --> (https://github...)